static/usr/share/subiquity/console-conf-write-login-details

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+pretty_name="$( awk -F "=" '$1 = /PRETTY_NAME/ {print $2;}' /usr/lib/os-release | sed 's/"//g')"
+tty_name=$(tty | cut -c 6-)
+
+get_first_ip() {
+  local ip_address
+  for iface in $(ip -br address | awk '{print $1}')
+  do
+    if test -n "${iface}" && ip addr show dev "${iface}" >/dev/null 2>&1; then
+        ip_address="$(ip -4 -br addr show dev "${iface}" | awk '{print$3}' | sed 's|\(.*\)/.*|\1|')"
+        if [ -n "${ip_address}" ]; then
+          echo "${ip_address}"
+          return 0
+        fi
+    fi
+  done
+  echo "<no ip address>"
+  return 2
+}
+
+print_ssh_commands() {
+  local user_name
+  # take any user with non zero
+  for h in /root $(getent passwd | grep -v -e '/nologin' -e '/false' | cut -d: -f6 | grep /home)
```

Comment:
Remove `/root` and remove the `grep /home`?

Reply:
We actually have this weird entry in passwd which would show up otherwise:
```
sync:x:4:65534:sync:/bin:/bin/sync
```

Reply:
That is a deprecated user with no real use. We should remove that user.

Reply:
Hm, I'll do that in a followup

Reply:
Yep.